labgavin/src/component/dashboard-container/index.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import './_dashboard-container.scss';
+import {connect} from 'react-redux';
+import {
+  categoryCreate,
+  categoryUpdate,
+  categoryDelete,
+} from '../../action/category-actions';
+import CategoryForm from '../category-form';
+import CategoryItem from '../category-item';
+
+class DashboardContainer extends React.Component {
+
+  componentDidMount() {
+    this.props.categoryCreate({title: 'Dining Out', budget: '500'});
+  }
+
+  render() {
+    return (
+      <main className="dashboard-container">
+        <h1>Track your Budget</h1>
+        <h2>Total Monthly Budget: {this.props.totalMonthly}</h2>
+        <h2>Total remaining: {this.props.totalRemaining}</h2>
+        <div className='categoryForm'>
+          <CategoryForm
+            buttonText='Create'
+            onComplete={this.props.categoryCreate} />
+        </div>
+        <div className="categoryList">
```

Comment:
kabob-case

Reply:
No, I'm allowed to write code how I like.

Reply:
Jk, Izzy is always right.